### ISSUE TITLE
Uknown operationId for any operation on Trappist

### ISSUE THREAD

**Issue by mutantcornholio** [polkadot-api/polkadot-api#515]
I'm getting following error just by calling `papi add -w wss://rococo-trappist-rpc.polkadot.io trappist`:

```
Uknown operationId on {
  event: 'operationCallDone',
  operationId: '1',
  output: '0x080e0000000f000000'
}
```

I'm also getting it on the normal startup of the faucet app.

**Comment by josepot** [comment]
Yeah, that's not really an issue anymore. I will get rid of those messages.

**Comment by mutantcornholio** [comment]
The thing is, `papi add` hangs forever. Does it reproduce on your end?

**Comment by mutantcornholio** [comment]
It feels like it's not connected to the previous cases. Here the log sample from faucet startup, and `operationCallDone` is actually the first mention of `operationId: 1`
[papi-debug.log](https://github.com/polkadot-api/polkadot-api/files/15321652/papi-debug.log)

**Comment by josepot** [comment]
In those logs we can clearly see that the messages are arriving out of order. Which means that either the RPC node that you are connecting to has a pretty old version of the Polkadot-SDK, or that there is a regression on the Polkadot-SDK which causes the messages to arrive out of order.

One way around this is would be to use the `--chainSpec` option. Eg:

```bash
papi add -c ./trappist-chainspec.json trappist
```

**Comment by mutantcornholio** [comment]
It can easily be an old version, given how low is the activity there.  
Is it possible to do (some kind of) a version check that would produce a meaningful error in this case?
